src/components/IframeComponent/index.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div v-if="loading">
+    <loader />
+  </div>
+  <div class="app">
+    <div class="header" :class="{ 'menu-hidden': !mainMenuVisible }">
+      <main-menu :main-menu-visible="mainMenuVisible" @toggle="toggleMainMenu" :subMenuVisible="false" />
+      <menu-bar :menu-bar-visible="mainMenuVisible" @toggle="toggleMainMenu" />
+    </div>
+    <!-- content -->
+    <div class="iframe-container">
+      <object :data="`${Experience.url}`" width="100%" height="100%" @load="onObjLoad()"></object>
```

Comment:
Please follow camelCase standards for the variable name.

Reply:
Done